stream_framer.go

```diff
@@ -113,8 +113,11 @@ func (f *streamFramer) maybePopNormalFrames(maxTotalLen protocol.ByteCount) []*w
 		}
 		id := f.streamQueue[0]
 		f.streamQueue = f.streamQueue[1:]
-		str, err := f.streamGetter.GetOrOpenSendStream(id)
-		if err != nil { // can happen if the stream completed after it said it had data
+		// This should never return an error.
+		// The stream will only be in the streamQueue, if it enqueued itself there.
+		str, _ := f.streamGetter.GetOrOpenSendStream(id)
```

Comment:
Yes, but this guarantee or the way we use the streamsQueue might change in the future – and we probably won't notice it at that point. Checking the error doesn't cost much, so I'd rather keep it in.